WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadProgressViewModelTest.kt

```diff
@@ -13,13 +13,15 @@ import org.wordpress.android.R.string
 import org.wordpress.android.TEST_DISPATCHER
 import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.test
+import org.wordpress.android.ui.jetpack.backup.download.progress.BackupDownloadProgressStateListItemBuilder
 import org.wordpress.android.ui.jetpack.backup.download.progress.BackupDownloadProgressViewModel.UiState.Content
 import org.wordpress.android.ui.jetpack.backup.download.progress.BackupDownloadProgressViewModel
 import org.wordpress.android.ui.jetpack.backup.download.progress.BackupDownloadProgressViewModel.UiState
 import org.wordpress.android.ui.jetpack.backup.download.usecases.GetBackupDownloadStatusUseCase
 import org.wordpress.android.ui.pages.SnackbarMessageHolder
 import org.wordpress.android.ui.utils.UiString.UiStringRes
 import org.wordpress.android.viewmodel.Event
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ProgressState
```

Comment:
Minor 🔍: Can you run "Optimise Imports"? I see a warning here.

Reply:
In 8ae41cc